.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,26 @@
+name: Benchmarking
+on:
+  - workflow_dispatch
+jobs:
+  benchmark:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        file:
+          - managed-1
+          - managed-2
+          - shared-1
+          - shared-2
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: npm
+      - run: npm ci
+      - run: npm run benchmark --silent -- -c 2 -n 10 -f analysis/hosting-stack/${{ matrix.file }}.txt -o csv > ${{ matrix.file }}.csv
```

Comment:
Instead of using specific files here, would it be possible to provide the files somehow when dispatching the workflow?

If possible, I would prefer not to include the files with the concrete URLs in the repo. Would be great if those could be provided on the fly, not in source code.

Reply:
Yes, I agree with this, but unfortunately this is not possible. Probably we should remove this workflow and URL files, right?

Reply:
@eugene-manuilov Agreed, in that case probably this PR should only have the benchmark command changes.

Can you move this workflow file (not the URL lists) into a public GitHub Gist on your account maybe? If you do that, I think it would be great to keep a reference to that in here, for example you could add a code comment somewhere in the `benchmark-url.js` file that links to this Gist (e.g. "Example for how to use this command in a GitHub workflow: {url}").

Reply:
Ok, removed.